lib/problem_check/twitter_login.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ProblemCheck::TwitterLogin < ProblemCheck
+  self.perform_every = 24.hours
+
+  def call
+    return [] if !authenticator.enabled?
```

Comment:
Related to your initial PR, what if we made `problems = check.call` into `problems = Array.wrap(check.call).compact`, then we can just do `return if` in these problem classes?

An even nuttier idea would be to make a specific `NoProblem` class that has a `nil?` method that returns `true`, then you can do the same compact thing. The only benefit to this would be some explicitness in the problem classes.

Reply:
I've been thinking about some accessory methods like `no_problem` and `problem`, but still fermenting. 🙂